test/verify/check-system-services

```diff
@@ -33,7 +33,7 @@ class TestServices(MachineCase):
         # Make sure the system finishes "booting" so that
         # when we add additional services to this target below
         # we don't race with the boot process
-        self.machine.execute("systemctl start default.target")
+        self.machine.execute("while ! systemctl is-active default.target; do sleep 1; done")
```

Comment:
BTW, the commit message "Actually wait until default.target is active" is a bit misleading. `systemctl start` *does* wait for default.target to become active. The "problem" is that it also starts it if it wasn't activating yet. That's the bit that we don't want, as it hides bugs.

Reply:
I changed the message